### PR TITLE
Don't overwrite xlsx files 

### PR DESCRIPTION
**Pull request by garethbowen**
This allows configurers to modify the xlsx files and not have them
overwritten the next time they run the convert.

Also restructures the test data to keep expected xmls separate to
allow for testing this case.

medic/medic#5884

### CODE REVIEW

File: src/lib/sync-fs.js [hunk 1]
```diff
@@ -74,8 +74,14 @@ function withoutExtension(fileName) {
   return extensionStart === -1 ? fileName : fileName.substring(0, extensionStart);
 }
 
+function copy(from, to, { overwrite=true }={}) {
+  if (overwrite || !fs.existsSync(to)) {
+    fs.writeFileSync(to, fs.readFileSync(from));
```
Comment (kennsippell) [inline src/lib/sync-fs.js]: Possibly cleaner with [fs.copyFileSync](https://nodejs.org/api/fs.html#fs_fs_copyfilesync_src_dest_flags)?

File: test/fn/convert-forms.utils.js [hunk 2]
```diff
@@ -16,23 +16,19 @@ module.exports = {
...
 
-          const generatedXml = expectedXml.replace(/\.expected\.xml$/, '.xml');
-
-          it(`should generate ${generatedXml} as expected`, () => {
-
-            assert.ok(fs.exists(generatedXml), `Missing generated XML file: ${generatedXml}`);
-            assert.equal(fs.read(generatedXml), fs.read(expectedXml), `Content of ${generatedXml} was not as expected.`);
-
-          });
+        const generatedXml = expectedXml.replace(/\/expected/, '');
```
Comment (kennsippell) [inline test/fn/convert-forms.utils.js]: Regex seems unrequired now. Can we replace `'/expected/'` with `'/'`?

File: src/lib/sync-fs.js (hunk 1 above)
Comment (garethbowen) [reply src/lib/sync-fs.js]: Definitely!

File: test/fn/convert-forms.utils.js (hunk 2 above)
Comment (garethbowen) [reply test/fn/convert-forms.utils.js]: Done.

**Pull request merged**